### PR TITLE
[WIP] Change version tags to fix CVEs

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**

This PR is related to meshery/meshery#4642

**Notes for Reviewers**
Still WIP and subject to tests

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you for this, @alphaX86. I think we're ready for the bump in Alpine and _might be_ ready for the bump in Golang. @meshery/adapter-maintainers thoughts?

**Comment by leecalcote** [comment]
@Revolyssup will you see the check in this PR are re-run after you have added the integration test workflows to this repo?

**Comment by leecalcote** [comment]
@alphaX86, preemptively, will you open a similar PRs on the other adapter repos?

**Comment by alphaX86** [comment]
> @alphaX86, preemptively, will you open a similar PRs on the other adapter repos?

It'll be done @leecalcote 👍

File: Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ COPY nginx/ nginx/
 # Build
 RUN GOPROXY=https://proxy.golang.org,direct CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -ldflags="-w -s -X main.version=$VERSION -X main.gitsha=$GIT_COMMITSHA" -a -o meshery-nginx-sm main.go
 
-FROM alpine:3.14 as jsonschema-util
+FROM alpine:3.15 as jsonschema-util
 RUN apk add --no-cache curl
 WORKDIR /
 RUN curl -LO https://github.com/layer5io/kubeopenapi-jsonschema/releases/download/v0.1.2/kubeopenapi-jsonschema
```
Comment (Aisuko) [inline Dockerfile:22]: Could we add the environment in order to save the version rather than directly to use it?

**Review by Aisuko — DISMISSED** [review]
Thanks, a suggestion please see below.

File: Dockerfile (hunk 1 above)
Comment (alphaX86) [reply Dockerfile:22]: I'll give a look and add 👍

**Review by Revolyssup — APPROVED** [review]

**Pull request merged**